### DISCUSSION TITLE
Keybinding for showing waypoints

### DISCUSSION THREAD

**Discussion by JustDen1234** [Courseplay/courseplay#7388] [Ideas]
When using CP, many times i want to take a quick look at the course itself. Therefore I need to open the HUD, click on the setting tab, and click one or more times on 'Show waypoints'. 
It would help me if there is a keybinding posibillity to cycle to the different options (like you do with clicking on 'Show waypoints).

**Comment by Tensuko** [comment]
Should be possible.
Will have a look at this.
